apps/nowcasting-app/pages/api/auth/[...auth0].ts

```diff
@@ -43,8 +43,9 @@ export default withSentry(
     },
 
     async logout(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
hey @braddf, I was working on getting the logout to work again since it's not working on the deployed `delta view` branch and this worked locally, but still not working on the deployed branch. on the vercel branch I get something like this when I click `sign out`. 
<img width="1256" alt="Screenshot 2023-01-31 at 17 44 25" src="https://user-images.githubusercontent.com/86949265/215825469-f483ffdb-8bc8-42a5-a740-56e9a818f75f.png">

Reply:
I _think_ this should be solved with the new Vercel ENV VAR in `.env.production` but let's def check on this on delta/staging 👍